nautobot/extras/jobs.py

```diff
@@ -1151,16 +1153,34 @@ def run_job(self, job_class_path, *args, **kwargs):
         raise KeyError(f"Job class not found for class path {job_class_path}")
     job = job_class()
     job.request = self.request
+    job_result = JobResult.objects.get(id=self.request.id)
+    payload = {
+        "job_result_id": self.request.id,
+        "job_name": job.name,
+        "user_name": job_result.user.get_full_name(),
+        "job_kwargs": kwargs,
```

Comment:
Looking at the payload, I think we would want to omit `job_kwargs` in the case of `job_model.has_sensitive_variables == True`?